qemu/tests/fio_perf.py

```diff
@@ -81,6 +81,32 @@ def get_version(session, result_file, kvm_ver_chk_cmd, guest_ver_cmd, type, driv
         result_file.write("### guest-kernel-ver : Microsoft Windows [Version ide driver format]\n")
 
 
+def clean(session, check_install_fio, tarball, os_type, guest_result_file, fio_path, timeout):
+    """
+    del temporary files in guest
+
+    :params session: VM session
+    :params check_install_fio: check if fio is installed in guest
+    :tarball: fio tar package
+    :param os_type: guest typet
+    :guest_result_file: fio result file in guest
+    :fio_path: path of fio source code
+    :param timeout: Timeout in seconds
+    """
+
+    if not session.cmd_status(check_install_fio):
+        if os_type == "linux":
+            session.cmd("rm -rf %s /tmp/%s %s" % (guest_result_file, os.path.basename(tarball), fio_path), timeout)
+        elif os_type == "windows":
+            session.cmd("del /f/s/q %s && rd /s/q %s" % (guest_result_file, fio_path), timeout)
+
+
+def uninstall_fio():
```

Comment:
Comments should start with a # and a single space.
Please remove the blank line between comments and `pass`